### PR TITLE
Bug: [Dev-8568] COVE: Dashboard Filter Values Not Updating From Data Change

### PR DESCRIPTION
**Pull request by mpallansch**

### CODE REVIEW

File: packages/dashboard/src/CdcDashboard.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import defaults from './data/initial-state'
 import { processData } from './helpers/processData'
 import { getVizKeys } from './helpers/getVizKeys'
 import { processDataLegacy } from './helpers/processDataLegacy'
+import { addValuesToSharedFilters } from './helpers/addValuesToSharedFilters'
```
Comment (joshlacey) [inline packages/dashboard/src/CdcDashboard.tsx:9]: can we merge dev-8191 before this ticket? I've moved the addValuesToSharedFilters function. Also in packages/core/components/Filters.tsx in that PR I have added this addValuesToSharedFilters function so maybe that will resolve this bug?

**Comment by adamdoe** [comment]
Waiting on 8191 to be merged.

**Comment by adamdoe** [comment]
@mpallansch now that 8191 is merged, can you retest this and catch your branch up? Thanks!

**Comment by mpallansch** [comment]
> @mpallansch now that 8191 is merged, can you retest this and catch your branch up? Thanks!

The branch is caught up now, I believe it still needs to be merged in, the test case is working in my branch and not on dev

**Pull request merged**